src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -26479,4 +26481,166 @@ private static final void registerNEIPlanetRecipes() {
                     24);
         }
     }
+
+    private static final void registerOpticalComponentRecipes() {
+        // Optical Boule
+        GT_Values.RA.addLaserEngraverRecipe(
+                new ItemStack[] {
+                    ItemList.Circuit_Silicon_Ingot5.get(1L), // Americium Boule
+                    GT_Utility.copyAmount(
+                            0L,
+                            GT_ModHandler.getModItem(
+                                    "bartworks", "gt.bwMetaGeneratedlens", 1L, 36)), // Magneto resonatic lens
```

Comment:
Can do Loader.isModLoaded("bartworks") and then use WerkstoffLoader.Material.get(OrePrefixes.lens, 0) Laser engravers don't consume lenses unless intended